examples/data-objects/table-document/src/table.ts

```diff
@@ -3,18 +3,14 @@
  * Licensed under the MIT License.
  */
 
-import { IFluidHandle } from "@fluidframework/core-interfaces";
 import { ICombiningOp, PropertySet } from "@fluidframework/merge-tree";
-import {
-    Jsonable,
-    JsonablePrimitive,
-} from "@fluidframework/datastore-definitions";
 
-export type TableDocumentItem = Jsonable<JsonablePrimitive | IFluidHandle>;
+export type TableDocumentItem = any;
 
 /**
  * @deprecated - ITable is an abandoned prototype.  Please use SharedMatrix with
```

Comment:
Is there a plan to drop this package from the repo?

Reply:
That is a good question.  I'll add it to my agenda for @curtisman to get his take on if it's okay to do so.